### PR TITLE
Added gradle support

### PR DESCRIPTION
**Pull request by dmisiuk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,5 +16,57 @@
...
+.idea/
+# if you remove the above rule, at least ignore the following:
+
+# User-specific stuff:
+# .idea/workspace.xml
+# .idea/tasks.xml
+# .idea/dictionaries
+
+# Sensitive or high-churn files:
+# .idea/dataSources.ids
```
Comment (oneils) [inline .gitignore]: we can  use pattern .idea/ to exclude the whole folder.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,11 @@
+### Gradle template
+.gradle
```
Comment (oneils) [inline .gitignore]: for excluding the whole folder we should add / to the end, e.g: 
.gradle/

File: .gitignore (hunk 1 above)
Comment (dmisiuk) [inline .gitignore]: Ok. I will change it. This file was generated by .gitignore plugin for idea.

File: .gitignore [hunk 3]
```diff
@@ -8,5 +16,57 @@
...
+## Plugin-specific files:
+
+# IntelliJ
+/out/
+
+# mpeltonen/sbt-idea plugin
+.idea_modules/
+
+# JIRA plugin
+atlassian-ide-plugin.xml
```
Comment (oneils) [inline .gitignore]: Do we really need this plugin in the project and as a result such exclusion ?

File: .gitignore [hunk 4]
```diff
@@ -8,5 +16,57 @@
...
+
+# Crashlytics plugin (for Android Studio and IntelliJ)
```
Comment (oneils) [inline .gitignore]: Do we really need this plugin in the project and as a result such exclusion ?

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,33 @@
+apply plugin: 'java'
+apply plugin: 'kotlin'
+apply plugin: 'spring-boot'
+
+repositories {
+    mavenCentral()
```
Comment (oneils) [inline build.gradle:6]: Also we can add a jFrog maven repository:
jcenter()

Comment (dmisiuk) [inline build.gradle:6]: done

**Pull request merged**